src/App.vue

```diff
@@ -71,17 +88,40 @@ body {
 }
 
 .scroll-container {
+  width: 400px;
```

Comment:
scroll container is supposed to be a utility class shared across the app for any view that's supposed to scroll. I didn't do a good job with putting it on the App component. It shouldn't have any non-scroll-related rules, like `width` or resizing.